### ISSUE TITLE
Basic authentication with passwords is deprecated

### ISSUE THREAD

**Issue by kzielin** [devmoath/jira-php#4]
Im unable to use package, because during creating client jira server responds with plain text "Basic authentication with passwords is deprecated.  For more information, see: https://developer.atlassian.com/cloud/confluence/deprecation-notice-basic-auth/ "

**Comment by devmoath** [comment]
Hi @kzielin,

Could you tell me what version of Jira you are using ?

The package targeting version 8 for now at least.

**Comment by kzielin** [comment]
I'm using Jira Cloud, so I assume, that they applying latest version.

**Comment by devmoath** [comment]
ok, I made this package on self-hosted Jira Server not Jira Cloud.

I will see how I can support the cloud version as well.

**Comment by devmoath** [comment]
After doing some research, I believe you can use your token as the password like so:

```php
$client = Jira::client('USERNAME', 'TOKEN', 'jira.domain.com');

$result = $client->issues()->search();

echo $result['issues'][0]['key']; // KEY-1000
```

give it a try and let me know if you still facing the issue.

**Comment by kzielin** [comment]
Yes! It is working like a charm. Thank You for this clarification :)

**Comment by devmoath** [comment]
Any time 😁
